wordpress-mobile/WordPress-Android
# Update WordPress screenshots

Fixes errors causing the screenshots to fail. 

1. view with id `scroll_view` no longer present when attempting to click "Block Posts" 
2. view with id `row_stats` no longer present, updated to use quick action button
2. API changes resulting in the need for new mocks

To test: 
* Run WPScreenshot test, test should pass. 
* Additionally you can test the lane `bundle exec fastlane screenshots` to check that screenshots are verified but I believe the [CI run](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-Android/23464/workflows/cd0ab704-a8b2-48a4-986d-1b9161717753) should be sufficient 


PR submission checklist:
